NuGet.Config

```diff
@@ -28,5 +28,6 @@
     <add key="nugetbuild" value="https://www.myget.org/F/nugetbuild/api/v3/index.json" />
     <add key="roslyn" value="https://dotnet.myget.org/F/roslyn/" />
     <add key="benchmarkdotnet-ci" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
+    <add key="arcade" value="https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json" />
```

Comment:
@mmitche can correct me if I'm wrong, but I believe that here you may want to use this feed instead: 
```xml
<add key="dotnet-tools" value="[https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json](https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json)" />
```

Reply:
Actually:
```
<add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```